frontend/src/types.ts

```diff
@@ -13,6 +13,7 @@ export interface IChain {
 }
 
 export interface IUser {
+  [key: string]: any;
```

Comment:
What is key here? An array, array of string? This seems like a strange type-signature for a key.

Reply:
I replied on my own review comment below because I need to highlight the code segment